### PR TITLE
fix: cros-keyboard-map

### PR DESCRIPTION
**Pull request by june-fish**
Will now actually apply keyboard map. Also needs: https://github.com/Ultramarine-Linux/packages/pull/51

### CODE REVIEW

**Comment by madonuko** [comment]
Why isn't this required by the UM sub-%package?

File: anda/system/cros-keyboard-map/cros-keyboard-map.spec [hunk 1]
```diff
@@ -53,15 +53,15 @@ mkdir -p %buildroot/etc/systemd/system
...
 
 [Service]
 Type=oneshot
 ExecStart=/bin/bash /usr/bin/um-generate-cros-keymap
+ExecStartPost=+/usr/bin/systemctl restart keyd.service
 
 [Install]
-WantedBy=sysinit.target
+WantedBy=multi-user.target
 EOF
```
Comment (madonuko) [inline anda/system/cros-keyboard-map/cros-keyboard-map.spec]: 1. wrong place to install this service to imo (`%_unitdir`)
2. don't like how this doesn't satisfy [-25](https://linear.app/fyra/issue/FYRA-25/move-in-tree-sources-into-seperate-repos)

File: anda/system/cros-keyboard-map/cros-keyboard-map.spec [hunk 2]
```diff
@@ -53,15 +53,15 @@ mkdir -p %buildroot/etc/systemd/system
 tee %buildroot/etc/systemd/system/cros-keyboard-map.service <<EOF
 [Unit]
 Description=Generate chromebook keyboard layout
-Before=keyd.service
 After=tmp.mount
 
 [Service]
 Type=oneshot
 ExecStart=/bin/bash /usr/bin/um-generate-cros-keymap
```
Comment (madonuko) [inline anda/system/cros-keyboard-map/cros-keyboard-map.spec]: Is this UM specific? why `um-`?

File: anda/system/cros-keyboard-map/cros-keyboard-map.spec [hunk 3]
```diff
@@ -53,15 +53,15 @@ mkdir -p %buildroot/etc/systemd/system
...
+ExecStartPost=+/usr/bin/systemctl restart keyd.service
```
Comment (madonuko) [inline anda/system/cros-keyboard-map/cros-keyboard-map.spec]: `keyd reload`?

File: anda/system/cros-keyboard-map/cros-keyboard-map.spec (hunk 2 above)
Comment (nothingneko) [reply anda/system/cros-keyboard-map/cros-keyboard-map.spec]: this isn't necessarily UM specific, could be used on Fedora, UBlue, etc

File: anda/system/cros-keyboard-map/cros-keyboard-map.spec (hunk 3 above)
Comment (nothingneko) [reply anda/system/cros-keyboard-map/cros-keyboard-map.spec]: restart should do the same things in practice

**Review by madonuko — APPROVED** [review]

**Review by madonuko — APPROVED** [review]

**Comment by raboneko** [comment]
## 💔 Some backports could not be created

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|f39|[](https://github.com/terrapkg/packages/pull/1987)|
|✅|f40|[](https://github.com/terrapkg/packages/pull/1988)|
|❌|f41|Backport failed because of merge conflicts|

### Manual backport
To create the backport manually run:
```
backport --pr 1958
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/10543135618) for details

File: anda/system/cros-keyboard-map/update.rhai [hunk 4]
```diff
@@ -1,7 +1,9 @@
 if filters.contains("nightly") {
+  rpm.global("tree_commit", gh_commit("WeirdTreeThing/cros-keyboard-map"));
   rpm.global("commit", gh_commit("WeirdTreeThing/cros-keyboard-map"));
   if rpm.changed() {
     rpm.release();
-    rpm.global("commit_date", date());
+    rpm.global("tree_commit_date", date());
+    rpm.global("um_commit_date", date());
```
Comment (madonuko) [inline anda/system/cros-keyboard-map/update.rhai:7]: btw this is the same

**Pull request merged**